react/index.js

```diff
@@ -369,7 +330,7 @@ const MiniCart = ({
       }
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
not related to this PR, but why this disabling? if necessary, maybe comment on why it was done as to prevent it from being removed

Reply:
I don't know why that is in there.